tests/specs/tsconfig.ts

```diff
@@ -263,6 +263,78 @@ export default testSuite(({ describe }) => {
 				const code2 = await fixture.readFile('dist/index2.js', 'utf8');
 				expect(code2).toMatch('__publicField(this, "foo", 100);');
 			});
+
+			test('ignores tsconfig.json in third party dependencies', async () => {
+				await using fixture = await createFixture({
+					// Fake external dependency
+					node_modules: {
+						'fake-lib': {
+							'index.js': 'export function testFn() { return "Hi!" }',
```

Comment:
Can you make this a `.ts` file to confirm TS files in node_modules can still be compiled?

Reply:
Done!